qiskit/circuit/library/__init__.py

```diff
@@ -17,9 +17,45 @@
 
 .. currentmodule:: qiskit.circuit.library
 
-Standard Gates
+The circuit library is a collection of well studied circuits, directives, and gates that can be plugged
+into many experiments allowing users to program at higher levels of abstraction when developing and
+experimenting with circuits.
+
+
+Any element of the circuit library can be attached to a circuit with the method
+:meth:`.QuantumCircuit.append`. For example, to append a multi-controlled CNOT:
+
+.. jupyter-execute::
+
+    from qiskit.circuit.library import MCXGate
+    gate = MCXGate(4)
+
+    from qiskit import QuantumCircuit
+    circuit = QuantumCircuit(5)
+    circuit.append(gate, [0, 1, 4, 2, 3])
+    circuit.draw('text')
```

Comment:
Is there a reason to use `circuit.draw('text')` here vs `print(circuit)` vs `circuit.draw('mpl')`

Reply:
`print(circuit)` is not exactly the same as `draw('text')` (see https://github.com/Qiskit/qiskit-terra/pull/5030). After a quick grep, the most common pattern seems `draw()`. I'm okey with any.

Reply:
I prefer the `mpl` drawer because of the pretty colors :P

Reply:
I cannot argue against that :) Fixed in e5c3d891e